src/main/java/org/apache/mesos/offer/CreateOfferRecommendation.java

```diff
@@ -17,9 +16,10 @@
 
     public CreateOfferRecommendation(Offer offer, Resource resource) {
         this.offer = offer;
-        this.operation = new OperationBuilder()
+        this.operation = Operation.newBuilder()
                 .setType(Operation.Type.CREATE)
-                .setCreate(Arrays.asList(resource))
+                .setCreate(Operation.Create.newBuilder()
+                        .addAllVolumes(Arrays.asList(Resource.newBuilder(resource).build())))
```

Comment:
Can't we still just pass `resource` instead of `Resource.newBuilder(resource).build()`? It's not going to get mutated.